reentr.c

```diff
@@ -365,16 +366,26 @@ Perl_reentrant_retry(const char *f, ...)
     void *retptr = NULL;
     va_list ap;
 
+    I32 key;
+
 #ifdef USE_REENTRANT_API
 
     dTHX;
 
+    key = Perl_keyword (aTHX_ f, strlen(f), FALSE /* not feature enabled */);
+
     /* Easier to special case this here than in embed.pl. (Look at what it
        generates for proto.h) */
     PERL_ARGS_ASSERT_REENTRANT_RETRY;
 
 #endif
 
+    if (key == 0) {
+    }
```

Comment:
Is something missing here?  The following `if (key < 0)` excludes key ==0 so this has no effect.

Reply:
A subsequent commit adds the missing stuff.  I could rebase so this doesn't appear until then.  It looks like:

     if (key == 0) {
 383a384,393
 > 
 > #ifdef HAS_GETSPNAM_R
 > 
 >         /* This is a #define as has no corresponding keyword */
 >         if (strEQ(f, "getspnam")) {
 >             key = KEY_getspnam;
 >         }
 > 
 > #endif